python/mlcroissant/pyproject.toml

```
@@ -15,7 +15,7 @@ authors = [
   { name = "Pierre Ruyssen" },
   { name = "Prabhant Singh" },
 ]
-
+requires-python = ">= 3.10"
```

```suggestion
requires-python = ">= 3.10, <=3.12"
```

Support in Python 3.13 maybe limited, for the current codebase, due to the use of aifc and audioop which are deprecated and to be removed in 3.13.

See https://peps.python.org/pep-0594/#deprecated-modules